src/UI/Presenter/PagePresenter.php

```diff
@@ -301,6 +301,28 @@ private function assignBasicSmartyVariables(): void
         $this->smarty->assign('validLogin', $gValidLogin);
         $this->smarty->assign('debug', $gDebug);
         $this->smarty->assign('registrationEnabled', $gSettingsManager->getBool('registration_enable_module'));
+        
+        // Design variables
+        $this->smarty->assign('additional_styles_file',  $gSettingsManager->getString('additional_styles_file'));
```

Comment:
All Admidio variables within the Smarty template should have the camel style e.g. `additionalStylesFile`

Reply:
Fixed (all three variales).